packages/calcite-components/src/components/alert/alert.e2e.ts

```diff
@@ -189,6 +202,76 @@ describe("calcite-alert", () => {
     expect(await alert3.isVisible()).toBe(true);
   });
 
+  it("should queue alerts", async () => {
```

Comment:
Nice! We could test that multiple of each queue value are added in the correct order (ensure a second "immediate" is shown, or multiple "next" are ordered as expected, etc).

Reply:
ok ill add that to the test. 👍

Reply:
beefed up the tests a little. 🐮